### PR TITLE
fix issues from dogfood feeding

### PR DESCRIPTION
**Pull request by 66eli77**

### CODE REVIEW

**Comment by DXCanas** [comment]
alternatively, couldn't we just have a `v-else` for when we don't have a thumbnail, fix a height, and fill it with black?

**Comment by 66eli77** [comment]
the v-else is currently used by showThumbnail to display the truncated card.

**Comment by 66eli77** [comment]
also the pure black default thumbnail might not be ideal, we may want to display some other image for easy identify the lack of thumbnail from the client side

File: kolibri/plugins/learn/assets/src/vue/search-widget/index.vue [hunk 1]
```diff
@@ -237,6 +237,7 @@
     background-color: $core-bg-canvas
     position: fixed
```
Comment (DXCanas) [inline kolibri/plugins/learn/assets/src/vue/search-widget/index.vue:238]: Didn't see that this was fixed, but I feel like this is something that should be avoided as often as possible.

Comment (66eli77) [inline kolibri/plugins/learn/assets/src/vue/search-widget/index.vue:238]: This fixed the issue on chrome, haven't tested on other browser s yet .
Avoid position:fixed ? For what reason ?

**Comment by indirectlylit** [comment]
This looks good to me - merged in. I haven't checked in detail for bugs/cross-browser issues though so we still need to do that.

@DXCanas care to elaborate on `fixed`? In general I agree that `absolute` is preferable because it makes the component re-usable (positioned relative to a parent rather than the viewport). However for the sidebar it seems reasonable to me?

**Comment by DXCanas** [comment]
@indirectlylit I didn't mean the sidebar in particular, it was just a general note about being conservative with it. I should have been clearer about that. The reason you mentioned is essentially the same for me -  fixed creates complications and often unpredictable behavior for the elements that life below it on the page.

**Pull request merged**